Specs/DateTime/French/DateTimeModel.json

```diff
@@ -2465,5 +2465,121 @@
         "End": 20
       }
     ]
+  },
+  {
+    "Input": "Je vais partir pour 3 heures",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "3 heures",
+        "TypeName": "datetimeV2.duration",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "PT3H",
+              "type": "duration",
+              "value": "10800"
+            }
+          ]
+        },
+        "Start": 20,
+        "End": 27
+      }
+    ]
+  },
+  {
+    "Input": "Je vais partir pour une durée de 3 heures",
```

Comment:
Can you add two more cases?
One sentence that only has a time entity using "heures" and another one that has both a time and a duration.

Reply:
cases added